Profiles - ZIB 2017/zib-ProcedureRequest.xml

```diff
@@ -295,11 +295,6 @@
         </discriminator>
         <rules value="open" />
       </slicing>
-      <mapping>

```

Comment:
Because I doubted the removal of this mapping (since it's also present in zib-Procedure) I discussed with Niek and we concluded it's better to keep this mapping here (even though you could argue it would be more appropriate to add this mapping to `.occurrenceDateTime` instead). Could you please revert this removal? ;)